### DISCUSSION TITLE
Why I found error in using PCL built by vcpkg?

### DISCUSSION THREAD

**Discussion by Bzz2745** [microsoft/vcpkg#29939] [Q&A]
Error info:
![image](https://user-images.githubusercontent.com/68807654/222031469-47433826-8019-4788-a0aa-92d4a1c69e32.png)
Header files used:
#include<pcl/io/pcd_io.h>
#include<pcl/point_types.h>
#include<pcl/point_cloud.h>
Error info in concept_check.hpp:
![image](https://user-images.githubusercontent.com/68807654/222031654-00fe7e20-a87f-41d7-bcdf-38a5a72f8755.png)
PS:I have searched "BOOST_PP_IIF_BOOST_PP_BOOL_" in vcpkg/installed/x64-windows/boost but can't find anyone

OS:windows11
complier:MSVC,VC143 CUDA12

I wonder what should I do to solve this or what I have done is wrong?

**Comment by Osyotr** [comment]
https://developercommunity.visualstudio.com/t/IntelliSens-reports-many-errors-for-the-/10287480
